woocommerce/woocommerce-gateway-stripe
# Catch request failure errors

Fixes #2528 

## Changes proposed in this Pull Request:
Added try/catch block in some places where exception catches are missing. These missing exception catches could cause fatal errors in the application and make it visible in the UI.

## Testing instructions
- Block requests to Stripe by adding `127.0.0.1  api.stripe.com` in your `etc/hosts` file.
- Check the Stripe settings in the admin dashboard and make sure there is no visible fatal error in the UI.
- Go to the shop as a customer. Add a payment method and a subscription to your account. Check different pages of `My Account`, Cart, Checkout, etc and make sure there is no visible fatal error in the UI.
